python/saw/__init__.py

```diff
@@ -270,25 +321,28 @@ def llvm_verify(module: LLVMModule,
                                       assumptions=lemmas,
                                       contract=contract,
                                       exception=err)
-            abort_proof = True
+    # If something else went wrong...
+    except Exception as err:
+        __global_success = False
+        for view in __designated_views:
```

Comment:
What is it that makes the views "designated"? The name `designated_connection` uses it in the sense of "canonical", but here it seems to be those that the user selected. What about something like `__active_views` instead?